values.yaml

```diff
@@ -41,6 +41,7 @@ global:
     # namespace pools: define k8s namespaces in which Astronomer will manage Airflow deployments
     namespacePools:
       enabled: false
+      keeproleandrolebindings: false
```

Comment:
it can just be role: true or false it should be default to true

Reply:
here we are not keeping the existing roles we are creating an option to either allow creation of roles by platform or by customer and our naming

Reply:
@pgvishnuram one of the acceptance criteria says: 
Introduce a keep flag so already create role and rolebindings are not deleted and synchronize on a helm update/upgrade.

Reply:
why do we need a keep flag here rbac: true is default unless customer sets them to false so this is fully of customer responsibility to make revoke resources and the naming ```keeproleandrolebindings``` does make any for the customers who already has those rbac applied

Reply:
done, since by default we are keeping the value to true so customer when needs to recreate role can mark it as false